setup.py

```diff
@@ -26,9 +26,10 @@
         "Operating System :: OS Independent",
         "Development Status :: 1 - Planning",
     ],
+    py_modules=["_icepyx_version"],
```

Comment:
Could you briefly explain what this line does?

Reply:
This is actually needed for https://github.com/icesat2py/icepyx/pull/172/files#diff-09833c49a3ea43c9bf0ca7b7caf2c88d716a37348b41a7c543aabb344c5d9091R3 to work. Since `_icepyx_version.py` is created on the fly during build, we need to make it a module so it can be imported. Here's a full explanation on it: https://docs.python.org/3.8/distutils/setupscript.html#listing-individual-modules. Without this, I keep getting module not found error :grimacing:

Reply:
Borrowed it from this: https://github.com/psf/black/blob/master/setup.py#L63 :)

Reply:
Ah ok, so it just packages things in the [`icepyx/`](https://github.com/icesat2py/icepyx/tree/development/icepyx) folder and not the stuff in `.github/workflows`, `doc` and `examples`?

Reply:
That specifications are actually in the MANIFEST file. the py_modules just looks into the source package I think. `setuptools_scm` will create `_icepyx_version.py`. And the `py_modules` will allow for it to be imported.

Reply:
Thanks for the explanation, I think it looks good :)

Reply:
No problem! Anytime :smile: